src/main/main.c

```diff
@@ -234,7 +234,7 @@ void init(void)
     servoMixerInit(masterConfig.customServoMixer);
 #endif
 
-#ifdef SPEKTRUM_BIND
+#if defined(SERIAL_RX) && defined(SPEKTRUM_BIND)
```

Comment:
@AndersHoglund : Ideally target.h shall only define things. Not defining something means that it is either default (e.g. for pins) or unused (`USE_*`).
`#undef`ing something is target.h should not be necessary.

`#if (USE_X = "YES")` - Only advantage I see is that you can use `"YES"` as default